tests/test_mprester.py

```diff
@@ -26,27 +28,31 @@
 from mp_api.client import MPRester
 from mp_api.client.core.settings import MAPIClientSettings
 
-os.environ["MP_API_KEY"] = "test"
+FAKE_MP_API_KEY = "12345678901234567890123456789012"  # 32 chars
 
 
 @pytest.fixture()
 def mpr():
+    # Only mock the API key in GitHub CI environment,
+    # otherwise people cannot run tests locally as the key is invalid
+    if os.getenv("GITHUB_ACTIONS") is not None:
+        os.environ["MP_API_KEY"] = FAKE_MP_API_KEY
+
     rester = MPRester()
     yield rester
     rester.session.close()
 
 
-@pytest.mark.skipif(os.getenv("MP_API_KEY", None) is None, reason="No API key found.")
```

Comment:
I believe this skip mark should be removed now, as the `MP_API_KEY` setter is put inside the fixture